### ISSUE TITLE
Please return arc-green theme or add a "dimmed" version of the current dark theme, v1.22 dark contrast is unbearable

### ISSUE THREAD

**Issue by yurikus** [go-gitea/gitea#31308]
### Description

v1.22 changed the colors of the dark theme. The new background is almost black and new contrast creates serious eye strain. Please return and support the arc-green theme. 

EDIT: If supporting arc-green is not an option as mentioned below by silverwind, please add a "dimmed" version of the current dark theme, with contrast on par with the arc-green of v1.21.

### Screenshots

![image](https://github.com/go-gitea/gitea/assets/11010776/ddecbc01-321c-4517-a943-19e37dd27dec)
![image](https://github.com/go-gitea/gitea/assets/11010776/4418cc5e-b609-454b-9ac5-788a8acf8d9c)

### Gitea Version

1.22

### Can you reproduce the bug on the Gitea demo site?

Yes

### Browser Version

Firefox 126

**Comment by silverwind** [comment]
I don't think we'll be going back to green primary, but a new low-contrast dark theme could be supported, similar to GitHub's "Dark dimmed".

**Comment by yurikus** [comment]
Dimmed version would be fine, presently it's unusable for me without dark reader

**Comment by techknowlogick** [comment]
In case others arrive at this thread, some history around this change, is that the increased contrast was to improve accessibility. Although, a dimmed version is likely possible, as long as it isn't the only option (so folks can choose the option that works the best for them).

**Comment by yurikus** [comment]
> a dimmed version is likely possible, as long as it isn't the only option
Let's start with at least one dimmed option as presently there's none and UX worsened.

**Comment by silverwind** [comment]
Other users do appreciate the increased contrast, but I guess you can never make it right for everyone.

Also I think https://github.com/go-gitea/gitea/issues/30160 is a dependency if we want to keep our sanity while writing additional themes.

**Comment by zc-devs** [comment]
For `return arc-green theme` alone, you can put [theme-arc-green.css](https://github.com/go-gitea/gitea/raw/release/v1.21/web_src/css/themes/theme-arc-green.css) in `GITEA_CUSTOM/...` directory. Works like a charm for now.
[Customizing the look of Gitea](https://docs.gitea.com/administration/customizing-gitea#customizing-the-look-of-gitea).

**Comment by yurikus** [comment]
> For `return arc-green theme` alone, you can put [theme-arc-green.css](https://github.com/go-gitea/gitea/raw/release/v1.21/web_src/css/themes/theme-arc-green.css) in `GITEA_CUSTOM/...` directory. Works like a charm for now. [Customizing the look of Gitea](https://docs.gitea.com/administration/customizing-gitea#customizing-the-look-of-gitea).

A short-lived workaround, given that theme is out of support. Thank you regardless.

**Comment by silverwind** [comment]
> > For `return arc-green theme` alone, you can put [theme-arc-green.css](https://github.com/go-gitea/gitea/raw/release/v1.21/web_src/css/themes/theme-arc-green.css) in `GITEA_CUSTOM/...` directory. Works like a charm for now. [Customizing the look of Gitea](https://docs.gitea.com/administration/customizing-gitea#customizing-the-look-of-gitea).
> 
> A short-lived workaround, given that theme is out of support. Thank you regardless.

Yes, it may still mostly work for 1.22, but eventually it's bound to break when CSS color variables are added or renamed.

**Comment by lsd-techno** [comment]
Here is a temporary workaround for the current version 1.22.1. This includes all variables implemented in the latest theme:

https://github.com/lsd-techno/gitea-theme

**Comment by FrostKiwi** [comment]
> Here is a temporary workaround for the current version 1.22.1. This includes all variables implemented in the latest theme:
> 
> https://github.com/lsd-techno/gitea-theme

Love the `arc-green` colors.
Unfortunately, using this breaks all Syntax Highlighting. 
Broken in Markdown, Broken in the file views 😭

**Comment by lsd-techno** [comment]
> > Here is a temporary workaround for the current version 1.22.1. This includes all variables implemented in the latest theme:
> > https://github.com/lsd-techno/gitea-theme
> 
> Love the `arc-green` colors. Unfortunately, using this breaks all Syntax Highlighting. Broken in Markdown, Broken in the file views 😭

Thanks for pointing out, [code markup looks much pretty now](https://github.com/lsd-techno/gitea-theme/issues/1), but can't see file views problem.

**Comment by hermannoffen** [comment]
I'd like to hint to the fact that by changing the color from green to blue, we threw the _corporate design_ under the bus. Should've changed the logo color as well then.

**Comment by QrackEE** [comment]
> I'd like to hint to the fact that by changing the color from green to blue, we threw the _corporate design_ under the bus. Should've changed the logo color as well then.

Agreed - I loved the green-dominant consistent with logotype in grey-ish dark theme; I wish had noticed and raised the voice when PR for theme change was open... Hope our 'cup of tee' doesn't go the blue way, as I much prefer green tea ;)

**Comment by 24unix** [comment]
Why was it dropped at all? It was so unique, I miss the green. This blue stuff is so ordinary :-(

**Comment by sahps** [comment]
> Why was it dropped at all? It was so unique, I miss the green. This blue stuff is so ordinary :-(

Absolutely. Gitea is no longer distinguishable from other web clients. Just plain old blue like all others. It is a shame. The old theme should be an option still.

But we can put it back to how it was with custom themes!

I've managed to get the unwanted blue theme disabled on our instance, but even when the default theme is the green one, the logon screen is still blue, can this be changed?

Edit: Nevermind, I must not have restarted the instance following changing the default theme. It works :)

Edit2: I created the new theme by writing a short script. First taking the new and old and creating a mapping of colours from one to the other. Then updating the new theme with the old colours. I have this being run as part of the Dockerfile so it is created during updates.

**Comment by lsd-techno** [comment]
> Edit: Nevermind, I must not have restarted the instance following changing the default theme. It works :)
> 
> Edit2: I created the new theme by writing a short script. First taking the new and old and creating a mapping of colours from one to the other. Then updating the new theme with the old colours. I have this being run as part of the Dockerfile so it is created during updates.

You need to recreate some folder structure to display file viewer and editor colorized.
I did some description [here](https://github.com/lsd-techno/gitea-theme)

**Comment by lazynooblet** [comment]
This theme change should have been made as an additional theme. Forcing the default theme change feels hostile to me. I hate the new theme, I feel like I'm back on GitHub. The uniqueness of Gitea's green is what gave the application its underdog sparkle. Now it wants to conform and just be Yet Another GitHub Alternative. I really want the green back, but the additional decision to simply archive and not maintain the original theme is yet another blow. It means I've spent way too long trying to get other green themes to work.

> I think generally we agree that arc-green is a rather unconventional color theme and I don't like it very much.

I guess "we" means the maintainers, not the populace. But its their playground I guess.

**Comment by silverwind** [comment]
It shouldn't be too hard to re-implement arc-green as an additional theme, and I guess we could accept it as a non-default theme.

The default theme should be as color-neutral as possible imho, and for example I quite like the dark theme of [MDN](https://developer.mozilla.org/en-US/), almost monochrome with a very slight blue hint. Maybe I will update the default dark theme to be closer to that one.

**Comment by TheFpiasta** [comment]
The compleate Theme change is a creasy game changer. For some people it is a fresh. But without a fix solution to change it back it is not greate. A couble of teammembers has complained about the none changeable color change. We are considering downgrading Gitea again. lul. Oo

**Comment by silverwind** [comment]
It's human nature to oppose stylistic changes like these color changes, but you will get used to it pretty fast. I do consider adding a "soft dark" theme similar to GitHub's, and maybe we can even add an option for green links.

**Comment by silverwind** [comment]
More theme options are planned and will come eventually but it will require a redesign of the current theme system to have proper combinable theme variations.
